storage/tests/unit/test_blob.py

```diff
@@ -366,7 +366,7 @@ def test__make_transport(self, fake_session_factory):
 
     def test__get_download_url_with_media_link(self):
         blob_name = 'something.txt'
-        bucket = mock.Mock(spec=[])
+        bucket = _Bucket(name='IRRELEVANT')
```

Comment:
Please use a `mock.Mock`. We don't **need** special classes for just a single property. Instead:

```python
bucket = mock.Mock(name='IRRELEVANT', spec=['name'])
```